node/rpc/src/lib.rs

```diff
@@ -232,6 +233,8 @@ where
                 "Filecoin.NetAddrsListen",
                 net_api::net_addrs_listen::<DB, KS, B>,
             )
+            // Encryption
+            .with_method("Filecoin.EncUnlock", enc_api::enc_unlock::<DB, KS, B>)
```

Comment:
If these methods arent part of FIlecoin but Forest, i would rather call prefix them with `Forest.`, so we can better differentiate what's specifically ours and whats not.